### ISSUE TITLE
Build a central web presence

### ISSUE THREAD

**Issue by samccann** [ansible-community/community-team#157]

**Comment by samccann** [comment]
We need   a central web presence for the Ansible community. This web presence will be based on persona-based journeys that guide the reader in two ways:

- To find the information they are looking for 
- To guide them along an 'experience' journey from new user through more advanced user, to community participant, all the way to community/contributor leader.

**Comment by samccann** [comment]
Steps identified so far to accomplish this idea:

- [ ] Communicate website WG to community
- [ ] Decide on domain name for new community site.
- [ ] Finalize requirements for the new community site 
- [ ] Finalize plans  WRT Ansible.com and the community site
- [ ] Set up github repo for community site
- [ ] Set up  a static site generator for the community site
- [ ] Set up a demo area for community site for community input
- [ ] Ensure accessibility guidelines are met on community site
- [ ] Find a web designer/UX person for community site
- [ ] Finalize the personas that drive the community site
- [ ] Design the community front page based on journeys
- [ ] Evaluate existing community pages for addition to community site
- [ ] Design the docs landing page for all projects on community site
- [ ] Design clickable image to shows Ansible ecosystem
- [ ] Design subpages for community site.
- [ ]Provide test site for community review
- [ ] Permanent redirect the old docsite  to community site
- [ ] Redirect other Ansible.com pages  to the community site.
- [ ] Determine commit requirements for community site

**Comment by Andersson007** [comment]
A general question: does the start of the work on this website depend on the vote on the `Community Plans part 1 - DNS & Website` topic (which hasn't btw started yet)?
- If it does, we won't be allowed to start community advertisement before it happens:(
Would feel like we've already decided everything.
- If it doesn't depend on the vote (or we don't want it to depend), I think we shouldn't vote at all on Plans part 1 or at least emphasize that the vote is not about the new community site itself but about DNS for example, i.e. about the details. Otherwise, we can't start all community-involving tasks

**Comment by GregSutcliffe** [comment]
I take your point, but I don't think it's as binary as you make it out to be. To take a different example, no (sane) government would propose a policy it hasn't done the background research, due diligence & initial groundwork on. We absolutely can do the initial preparation while the votes progress - if they don't pass, we'll be scrapping months of work, so whats a few more days?

Assuming they do, then what we have prepared can be brought to the working group for further discussion, and can be changed if there is disagreement. We're not cutting people out of the loop, or deciding things a priori, we're just making a start.

**Comment by Andersson007** [comment]
SGTM, though it should be clearly mentioned (something like "As we see large support for the idea to have the new community site ..." not to provoke feelings like all those discussions with the community were formal

**Comment by cybette** [comment]
Right, we can also point out where we already have had the discussions (Cfgmgmtcamp, Contrib summit, online meetings, chats, github etc.) to establish that while those are not formal votes, they are important input as part of our due diligence and we have already been incorporating feedback along the way.

**Comment by GregSutcliffe** [comment]
I don't think that comment belongs on the vote itself, it muddles things up. I'll be happy to put something on the discussion thread along with a link to the vote, though.
